docs/sources/configuration/_index.md

```diff
@@ -276,11 +276,11 @@ The `querier` block configures the Loki Querier.
 # Maximum lookback beyond which queries are not sent to ingester.
 # 0 means all queries are sent to ingester.
 # CLI flag: -querier.query-ingesters-within
-[query_ingesters_within: <duration> | default = 0s]
+[query_ingesters_within: <duration> | default = 2h]
```

Comment:
Small suggestion, give `query_ingesters_within` some buffer on top of `max_chunk_age`, because the latter is parameterized by `last_line_timestamp - first_line_timestamp` and not necessarily `current_time`. Generally this will be a small value (seconds), but not giving a bit of buffer can result in regularly ignoring log lines on this border from queries.
```suggestion
[query_ingesters_within: <duration> | default = 3h]
```